src/components/Transaction/AccountBalance.js

```diff
@@ -0,0 +1,14 @@
+import {useParams} from 'react-router-dom';
+
+import useStore from '../../hooks/useStore';
+
+export default function AccountBalance() {
+	const {accountID} = useParams();
+	const getAccountWithData = useStore(state => state.getAccountWithData);
+	const account = getAccountWithData(accountID);
+	return (
+		<>
+			<h2>Current Balance: {account.value} €</h2>
+		</>
```

Comment:
You don't need the fragments here.
```suggestion
		<h2>Current Balance: {account.value} €</h2>
```